zapd/models.py

```diff
@@ -207,10 +211,6 @@ class AMWallet(db.Model):
     def __init__(self, address):
         self.address = address
 
-    @classmethod
-    def from_address(cls, session, address):
```

Comment:
you should not remove this code is it used in other parts

Reply:
accidentally removed it, i was supposed to remove from_device_id. placed from_address back,